lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -119,6 +119,8 @@
 
                     this.teamModel.save(data, { wait: true })
                         .done(function(result) {
+                            // So that the topics list can be refreshed.
+                            Backbone.trigger('teamModel:add', view.teamModel);
```

Comment:
I don't like manufacturing a fake event like this. I'd rather that the model be added to the collection because then anyone who is relying on the collection can know that it has been updated.